src/main/java/com/textmessenger/controller/UserController.java

```diff
@@ -29,23 +34,22 @@
 import javax.validation.Valid;
 import java.io.IOException;
 import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Function;
 
 @RestController
-@RequestMapping("/api/users")
+@RequestMapping(value = UserController.REQUEST_PATH_API_USERS)
 public class UserController {
+
+  static final String REQUEST_PATH_API_USERS = "/api/users";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Refactor your code to get this URI from a customizable parameter. ([squid:S1075](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS1075))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=236&issues=AWYK8i_OvOWp7Yd6HYt2&open=AWYK8i_OvOWp7Yd6HYt2)

[ISSUE_KEY]: AWYK8i_OvOWp7Yd6HYt2